specutils/tests/test_region_extract.py

```diff
@@ -18,10 +18,9 @@
 
 
 def test_region_simple(simulated_spectra):
-    np.random.seed(42)
 
     spectrum = simulated_spectra.s1_um_mJy_e1
-    uncertainty = StdDevUncertainty(0.1*np.random.random(len(spectrum.flux))*u.mJy)
+    uncertainty = StdDevUncertainty(0.1*np.ones(len(spectrum.flux))*u.mJy)
```

Comment:
You can further simplify the call with https://numpy.org/doc/2.0/reference/generated/numpy.full.html . This comment also applies to all similar occurrences in this PR.

```suggestion
    uncertainty = StdDevUncertainty(np.full(spectrum.flux.shape, 0.1) * u.mJy)
```